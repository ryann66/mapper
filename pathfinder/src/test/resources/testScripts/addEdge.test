CreateGraph g1
AddNode g1 p
AddNode g1 c
ListChildren g1 p
ListChildren g1 c
AddEdge g1 p c 1
ListChildren g1 p
ListChildren g1 c
AddEdge g1 p c 2
ListChildren g1 p
ListChildren g1 c
AddEdge g1 c p 3
ListChildren g1 p
ListChildren g1 c
#self connecting edges
AddNode g1 n1
AddEdge g1 n1 n1 4
ListChildren g1 n1
AddEdge g1 n1 n1 5
ListChildren g1 n1