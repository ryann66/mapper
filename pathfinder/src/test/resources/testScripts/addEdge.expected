created multigraph g1
added node p to g1
added node c to g1
the children of p in g1 are:
the children of c in g1 are:
added edge 1.000 from p to c in g1
the children of p in g1 are: c(1.000)
the children of c in g1 are:
added edge 2.000 from p to c in g1
the children of p in g1 are: c(1.000) c(2.000)
the children of c in g1 are:
added edge 3.000 from c to p in g1
the children of p in g1 are: c(1.000) c(2.000)
the children of c in g1 are: p(3.000)
#self connecting edges
added node n1 to g1
added edge 4.000 from n1 to n1 in g1
the children of n1 in g1 are: n1(4.000)
added edge 5.000 from n1 to n1 in g1
the children of n1 in g1 are: n1(4.000) n1(5.000)